Dockerfile

```diff
@@ -58,43 +63,49 @@ RUN set -x \
  && 2>/dev/null 1>&2 gem update --system --quiet \
  # install redmine
  && mkdir -p ${WORKDIR} \
- && curl -L http://www.redmine.org/releases/redmine-${REDMINE_VERSION}.tar.gz | tar xfz - --strip-components=1 -C ${WORKDIR} \
+ && mkdir -p /redmine_source \
+ && wget "https://www.redmine.org/releases/redmine-${REDMINE_VERSION}.tar.gz" \
+ && echo "${REDMINE_TARGZ_SHA256} *redmine-${REDMINE_VERSION}.tar.gz" | sha256sum -c - \
+ && tar -xf redmine-${REDMINE_VERSION}.tar.gz --strip-components=1 -C ${WORKDIR} \
+ && mv redmine-${REDMINE_VERSION}.tar.gz /redmine_source/redmine-${REDMINE_VERSION}.tar.gz \
+ && mkdir -p ${WORKDIR}/app/assets/config && touch ${WORKDIR}/app/assets/config/manifest.js \
  # set temporary database configuration for bundle install
- && DATABASE_TYPE=postgresql \
-    DATABASE_IP=localhost \
-    DATABASE_DB=redmine \
-    DATABASE_USER=redmine \
-    DATABASE_USER_PASSWORD=redmine \
-    eval "echo \"$(cat  ${WORKDIR}/config/database.yml.tpl)\"" > ${WORKDIR}/config/database.yml \
- # Install (available) rubycas-client version
- && git clone https://github.com/cloudogu/rubycas-client.git \
+ && cp ${WORKDIR}/config/database.yml.tpl ${WORKDIR}/config/database.yml \
```

Comment:
This seems redundant.

Reply:
Why?

Reply:
My error, i thought it is unnecessary since the config is not legit. But without the config, the postgresql bundle will not be installed.